app/content/profile/index.js

```diff
@@ -111,52 +296,88 @@ function Index(props) {
             ) : (
               <Text style={styles.noDrillsText}>No drills attempted yet</Text>
             )}
+
             <BottomSheetModal
               ref={bottomSheetModalRef}
               index={1}
               snapPoints={snapPoints}
               onChange={handleSheetChanges}
             >
-              <View>
+              <View style={styles.modalContent}>
+                {/* Close Button */}
                 <Pressable
-                  onPress={() => {
-                    bottomSheetModalRef.current.close();
-                  }}
-                  //width={"100%"}
-                  //alignItems={"center"}
+                  onPress={() => bottomSheetModalRef.current.close()}
+                  style={styles.closeButton}
                 >
-                  <Text
-                    style={{
-                      textAlign: "left",
-                      marginLeft: 5,
-                      fontSize: 15,
-                      color: "red",
-                    }}
-                  >
-                    Cancel
+                  <Text style={styles.closeButtonText}>Close</Text>
+                </Pressable>
+
+                {/* Profile Picture */}
+                <View style={styles.profilePictureContainer}>
+                  <TouchableOpacity onPress={handleImageClick}>
+                    <Image
+                      source={{ uri: userData.pfp }}
+                      style={styles.profilePicture}
+                    />
+                  </TouchableOpacity>
+                  <View style={styles.penIconContainer}>
+                    <TouchableOpacity onPress={handleImageClick}>
+                      <MaterialIcons name="edit" size={24} color="black" />
+                    </TouchableOpacity>
+                  </View>
+                </View>
+
+                <TextInput
+                  style={styles.input}
+                  value={name}
+                  onChangeText={handleNameChange}
+                  placeholder="Enter your name"
+                />
+                <TextInput
```

Comment:
remove this, we won't have email changing

Reply:
I on purpose left it for tomorrow's demo. Tomorrow, @solderq35 and I want to show it to the coach and ask him what he thinks about allowing them to update the user's email field.

Reply:
alright